browser/src/core/Socket.js

```diff
@@ -1520,6 +1520,15 @@ app.definitions.Socket = L.Class.extend({
 
 	_onSocketClose: function () {
 		window.app.console.debug('_onSocketClose:');
+		if (!this._map._docLoadedOnce) {
+			var postMessageObj = {
+				errorType: 'websocketconnectionfailed',
+				success: false,
+				errorMsg: errorMessages.websocketconnectionfailed
```

Comment:
missing `result` property:
```
success: <boolean>
result:  <string>
errorMsg: <string>
```
https://sdk.collaboraonline.com/docs/postmessage_api.html

Reply:
apart of that one thing looks good :)